qemu/qmp.go

```diff
@@ -746,7 +746,12 @@ func (q *QMP) ExecuteBlockdevDel(ctx context.Context, blockdevID string) error {
 		return q.executeCommand(ctx, "blockdev-del", args, nil)
 	}
 
-	args["id"] = blockdevID
+	if q.version.Major == 2 && q.version.Minor == 8 {
```

Comment:
Should this be >= ?

Reply:
No. If qemu version > 2.8, it will be processed by previous if statement to delete blockdev by `blockdev-del` qmp command,  while qemu 2.8 delete blockdev by `x-blockdev-del` qmp command.More details you can view this issue: https://github.com/kata-containers/runtime/issues/637

Reply:
@woshijpf Acknowledged.